src/style.scss

```diff
@@ -94,29 +94,28 @@ hr {
 }
 
 // Styling for dropdown links in the nav
-.dropdown-toggle {
-  color: #000 !important;
+.navbar-nav .nav-link.dropdown-toggle {
+  color: $black;
 }
```

Comment:
On hover and focus, the dropdown link changes to Bootstrap's selector and its grey color. This will fix it:
```
  &:hover, &:focus {
    color: $black;
  }
```

Reply:
Done!